person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_accessor :id, :name, :age
+
+  def of_age?
+    @age > 18
+  end
+
+  def can_use_services?
+    @age > 18 or @parent_permission == true
+  end
```

Comment:
- [ ] You are supposed to reuse your `of_age?` method here to keep your code **DRY**. Kindly fix that